### ISSUE TITLE
Getting error while installing on Windows

### ISSUE THREAD

**Issue by webticas** [coral-xyz/anchor#2004]
Getting following errors while using avm install latest and avm use latest commands:
![image](https://user-images.githubusercontent.com/71602752/175784319-5b5b64a5-88ca-4692-b05d-fe30904fa3ad.png)

**Comment by webticas** [comment]
> Hi, I'm facing the same problem, was wondering if you managed to find a solution?

Yes, I tried installing without avm and it worked. Use the command in 'Build from source for other operating systems without avm' and it should work.

**Comment by shravanandoria** [comment]
Can't set the avm version on windows 11
![image](https://github.com/coral-xyz/anchor/assets/36509067/5ba6b266-4518-4936-98bb-911a05b2cc4a)

**Comment by MorningLit** [comment]
no love for windows users? 😢

**Comment by acheroncrypto** [comment]
The issue is, some of the Solana tooling, such as the `solana-test-validator`, don't work in Windows. There is not much of a point of working on Windows support until those are fixed because Anchor CLI heavily depends on these tools for the development workflow.
